plugins/modules/files/iso_extract.py

```diff
@@ -101,7 +101,8 @@ def main():
             image=dict(type='path', required=True, aliases=['path', 'src']),
             dest=dict(type='path', required=True),
             files=dict(type='list', elements='str', required=True),
-            force=dict(type='bool', default=True, aliases=['thirsty']),
+            force=dict(type='bool', default=True, aliases=['thirsty'],
+                       deprecated_aliases=[dict(name='thirsty', version='3.0.0', collection_name='community.general')]),
```

Comment:
Please remove the below `deprecate()` call. Also, this change does not need a changelog fragment IMO, or only a `bugfixes:` entry with `use proper alias deprecation mechanism for ``thirsty`` alias of ``force``.`